_i18n/de/_posts/2021-02-15-white-impressions.md

```diff
@@ -1,10 +1,29 @@
 ---
 layout: post
-title:  "First"
-date:   2021-02-07 17:51:00 +0100
+title:  "Weiße Eindrücke"
+date:   2021-02-15 21:53:00 +0100
 categories: posts
 ---
 
-erster Beitrag.
+Ich lebe in Deutschland seit Jan/2016. 2 Tage nach ich nach Berlin gekommen bin, ich habe zum zweiten Mal in meinem Leben Schnee erlebt (das erster mal war in Turin - Italien). Ich erinnere mich wie glücklich und fasziniert ich war zu weiße Straßen sehen. Wie die Bäume, die vorher hätten keinen Blätter, jetzt wurden von einer dünnen und glatten Schneeschicht bedeckt. In 2016 gab es nicht so viele Schnee: aber es war genüg, um mir glücklich und erfreut zu machen.
 
+Mein 2021 hat (ein bisschen) ähnlich angefangen. Aber dieses mal, mit viel mehr Schnee! Am 2. Januar, ich bin in [Die Rhön](https://de.wikipedia.org/wiki/Rh%C3%B6n) mit meinem Freund und seiner Familie gegangen. Ich hatte keine Ahnung wie der Ort war: ich wusste ich wäre (hoffentlich) etwas Schnee in diesen Tag sehen.
 
+Ich bin an das warme und konstante Wetter aus Fortaleza, meiner Heimatstadt, gewöhnt. Also natürlich alles, was dem Schnee nahe kommt, packt meine Aufmerksamkeit.
+
+Also da war es: Bergen, Schnee und Kinder rutschen die Berghänge hinunter. Die Blicke waren atemberaubend und ich denke, das war die beste Weg, um das Jahr anfangen und die kälte Monate hier wahrhaftig genießen.
+
+An den gleichen Tag bin ich nach Berlin zurückgekommen und war bereit, um das neues Jahr zu beginnen. Die Woche vergingen und alles, was ich durch mein Wohnungsfenster sehe konnte, waren Tagen mit viele wolkige Himmel.
```

Comment:
dem gleichen Tag

bereit, dass neue Jahr

Reply:
Es ist wie "to" im Englischen, aber ich glaube nur bei bestimmten verben? Ich bin mir nicht ganz sicher was die genaue Regel ist.  hier ein paar Beispiele:
I went to Berlin, to have fun: Ich ging nach Berlin, um Spass zu haben.
I looked back, to see if the Duck followed me: Ich schaute zurueck, um zu sehen ob die Ente mir folgte.
I ran faster, to escape the duck: Ich rannte schneller, um der Ende zu entkommen.
Ich musste kämpfen, um zu leben. I had to fight to live